src/normalizations/normalize-calendar-events.ts

```diff
@@ -94,7 +94,7 @@ export const normalizeCalendarEvent = ({
 	const start = instanceStart || (invite as Invite)?.start?.u || appointment.inst?.[0]?.s;
 	const dur = (instance as ExceptionReference)?.dur ?? appointment.dur;
 	const user = getUserAccount();
-	const iAmOrganizer = appointment.or.a === user.name;
+	const iAmOrganizer = appointment?.or?.a === user?.name;
```

Comment:
The expression seems to be _true_ even when both operand are _undefined_. Is it a condition impossible to achive or an additional control  is needed?